app/MMB/src/scenes/tripServices/insurance/InsuranceTermsPdfScene.js

```diff
@@ -0,0 +1,23 @@
+// @flow strict
+
+import * as React from 'react';
+import { DeviceInfo } from '@kiwicom/mobile-localization';
+import { PdfViewAndStore } from '@kiwicom/mobile-pdf';
+
+const InsuranceTermsPdfScene = () => {
+  const languageCode = DeviceInfo.getLanguage();
+  let url =
+    'https://s3-eu-west-1.amazonaws.com/skypicker-pdf/insurance/VPPCP-Kiwi.com_ENG-CZE.pdf';
+  if (languageCode === 'cs') {
+    url =
+      'https://s3-eu-west-1.amazonaws.com/skypicker-pdf/insurance/VPPCP-Kiwi.com_CZE-ENG.pdf';
+  }
```

Comment:
this pdf is just available in two languages?

Reply:
That's what I found in the native repository, so I assume the answer is yes. I can ask again to the PM.

Reply:
Anyways, maybe it is even better to move these urls to the graphql, if some url changes, or there are added more, we would have to change this on all clients. Better to change on graphql only

Reply:
That makes sense, I will do that.

Reply:
@tbergq https://github.com/kiwicom/graphql/pull/280